packages/icons/public/index.html

```diff
@@ -5,12 +5,9 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Swiss Post Icons</title>
-    <link
-      rel="stylesheet"
-      href="https://unpkg.com/@swisspost/design-system-styles/basics.css"
-      integrity="sha384-EyL4eIC/kQ/za1rCC43o/z+2Flb/wou2q5pYH7DAg9UURlI7E5zt4vy4I2RJmZJI"
-      crossorigin="anonymous"
-    />
+    <link rel="stylesheet" href="https://unpkg.com/@swisspost/design-system-styles/basics.css" />
+    <!-- crossorigin="anonymous" -->
+    <!-- integrity="sha384-EyL4eIC/kQ/za1rCC43o/z+2Flb/wou2q5pYH7DAg9UURlI7E5zt4vy4I2RJmZJI" -->
```

Comment:
```suggestion
    <!--
      call the following url to get the latest integrity:
      https://unpkg.com/@swisspost/design-system-styles@latest/basics.css?meta
    -->
    <link
      rel="stylesheet"
      href="https://unpkg.com/@swisspost/design-system-styles/basics.css"
      integrity="sha384-lpjqj3vSaSVPowSj8Pz7soaXpsD9Fu7MsFwjJu/DPJKPsK7FRlHlZCNv+cPF01kY"
      crossorigin="anonymous"
    />
```

Reply:
Please add such a comment also for the alpinejs script at the bottom of this file.